### ISSUE TITLE
PostgreSQL query exception on user-related pages

### ISSUE THREAD

**Issue by k-popov** [open-metadata/OpenMetadata#6620]
**Affected module**
- UI
- backend (?)

**Describe the bug**
A pop-up is displayed with the following error messages:
```
An exception with message [org.postgresql.util.PSQLException: ERROR: subquery in FROM must have an alias Hint: For example, FROM (SELECT ...) [AS] foo. Position: 47 [statement:"/* UserDAO.listCount */ SELECT count(id) FROM (SELECT ue.id FROM user_entity ue LEFT JOIN entity_relationship er on ue.id = er.toId LEFT JOIN team_entity te on te.id = er.fromId and er.relation = :relation WHERE ue.deleted = FALSE AND ((ue.json#>'{isAdmin}')::boolean) = TRUE AND ue.json#>'{isBot}' IS NULL OR ((ue.json#>'{isBot}')::boolean) = FALSE ) AND (:team IS NULL OR te.name = :team) GROUP BY ue.id) subquery", arguments:{positional:{4:null,5:10}, named:{team:null,relation:10}, finder:[]}]] was thrown while processing request.

An exception with message [org.postgresql.util.PSQLException: ERROR: subquery in FROM must have an alias Hint: For example, FROM (SELECT ...) [AS] foo. Position: 47 [statement:"/* UserDAO.listCount */ SELECT count(id) FROM (SELECT ue.id FROM user_entity ue LEFT JOIN entity_relationship er on ue.id = er.toId LEFT JOIN team_entity te on te.id = er.fromId and er.relation = :relation WHERE ue.deleted = FALSE AND ue.json#>'{isBot}' IS NULL OR ((ue.json#>'{isBot}')::boolean) = FALSE ) AND (:team IS NULL OR te.name = :team) GROUP BY ue.id) subquery", arguments:{positional:{4:null,5:10}, named:{team:null,relation:10}, finder:[]}]] was thrown while processing request.
```
Also the following requests (seen in browser dev tools) result in described responses (all have 500 status code):
- https://openmetadata.example.com/api/v1/users?&isBot=false&limit=0
```
{"responseMessage":"An exception with message [org.postgresql.util.PSQLException: ERROR: subquery in FROM must have an alias\n  Hint: For example, FROM (SELECT ...) [AS] foo.\n  Position: 47 [statement:\"/* UserDAO.listCount */ SELECT count(id) FROM (SELECT ue.id FROM user_entity ue LEFT JOIN entity_relationship er on ue.id = er.toId LEFT JOIN team_entity te on te.id = er.fromId and er.relation = :relation  WHERE ue.deleted = FALSE AND ue.json#>'{isBot}' IS NULL OR ((ue.json#>'{isBot}')::boolean) = FALSE )   AND (:team IS NULL OR te.name = :team) GROUP BY ue.id) subquery\", arguments:{positional:{4:null,5:10}, named:{team:null,relation:10}, finder:[]}]] was thrown while processing request."}
```
- https://openmetadata.example.com/api/v1/users?&isAdmin=true&isBot=false&limit=0
```
{"responseMessage":"An exception with message [org.postgresql.util.PSQLException: ERROR: subquery in FROM must have an alias\n  Hint: For example, FROM (SELECT ...) [AS] foo.\n  Position: 47 [statement:\"/* UserDAO.listCount */ SELECT count(id) FROM (SELECT ue.id FROM user_entity ue LEFT JOIN entity_relationship er on ue.id = er.toId LEFT JOIN team_entity te on te.id = er.fromId and er.relation = :relation  WHERE ue.deleted = FALSE AND ((ue.json#>'{isAdmin}')::boolean)  = TRUE  AND ue.json#>'{isBot}' IS NULL OR ((ue.json#>'{isBot}')::boolean) = FALSE )   AND (:team IS NULL OR te.name = :team) GROUP BY ue.id) subquery\", arguments:{positional:{4:null,5:10}, named:{team:null,relation:10}, finder:[]}]] was thrown while processing request."}
```
- https://openmetadata.example.com/api/v1/users?fields=profile,teams,roles&&isBot=false&limit=12
```
{"responseMessage":"An exception with message [org.postgresql.util.PSQLException: ERROR: subquery in FROM must have an alias\n  Hint: For example, FROM (SELECT ...) [AS] foo.\n  Position: 47 [statement:\"/* UserDAO.listCount */ SELECT count(id) FROM (SELECT ue.id FROM user_entity ue LEFT JOIN entity_relationship er on ue.id = er.toId LEFT JOIN team_entity te on te.id = er.fromId and er.relation = :relation  WHERE ue.deleted = FALSE AND ue.json#>'{isBot}' IS NULL OR ((ue.json#>'{isBot}')::boolean) = FALSE )   AND (:team IS NULL OR te.name = :team) GROUP BY ue.id) subquery\", arguments:{positional:{4:null,5:10}, named:{team:null,relation:10}, finder:[]}]] was thrown while processing request."}
```

**To Reproduce**
- Install OpenMetadata 0.11.3 in Kubernetes with helm chart
- Configure OpenMetadata to use external (outside k8s cluster) PostgreSQL 14 database
- Set up custom OIDC (ADFS to be exact) with the following settings:
```
  authentication:
    provider: "custom-oidc"
    customOidcAuthenticationProviderName: ADFS
    publicKeys:
      - "https://example.com/adfs/discovery/keys"
    authority: "https://example.com/adfs"
    clientId: "aaaaaa-bbbbb-ccccc-dddd-valid-client-id"
    callbackUrl: "https://openmetadata.example.com/callback"
    jwtPrincipalClaims:
      - upn
      - unique_name
      - sub
```
- Authenticate and go to https://openmetadata.example.com/teams-and-users

**Expected behavior**
No exceptions happen.

**Version:**
 - OS: Linux
 - Python version: ?
 - OpenMetadata version: 0.11.3 (official Docker image)
 - OpenMetadata Ingestion package version: ?
 - PostgreSQL 14

**Additional context**
None

**Comment by vivekratnavel** [comment]
Hi @k-popov, this should have been fixed in the 0.11.4 release. Can you please update to 0.11.4 and check if you are still hitting this error?

**Comment by k-popov** [comment]
Just tried the official 0.11.4 image and it's fine, I don't see the issue any more.
P.S. Sorry for not checking for newer version before posting the bugreport.
